.github/workflows/tracing.yml

```diff
@@ -14,32 +14,52 @@ concurrency:
 jobs:
   macos:
     runs-on: macos-latest
+    env:
+      DD_API_KEY: ${{ secrets.DD_API_KEY_CI_APP }}
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/node/setup
       - run: yarn install
-      - run: yarn test:trace:core:ci
+      - run: MOCHA_FILE=./test-results-macos.yml yarn test:trace:core:ci
```

Comment:
Can we do instead `test-results/mocha/macos.xml`? (also note that it should be `xml` and not `yml`). This would be closer to the default, and the gitignore file can also be updated to add `test-results`.

Reply:
Actually the name of the test should probably also be in the name of the file or in the path since there could be more than 1 test running on macOS.